tensorflow_addons/optimizers/constants.py

```diff
@@ -0,0 +1,5 @@
+import tensorflow as tf
+
+BASE_OPTIMIZER_CLASS = tf.keras.optimizers.legacy.Optimizer
+if tf.__version__[:3] <= "2.8":
```

Comment:
As I wrote previously:
```
>>> tf.__version__
'2.10.0-dev20220531'
>>> tf.__version__[:3]
'2.1'
>>> tf.__version__[:3] > "2.8"
False
```

Please fix this condition (also in other files) or just check if `tf.keras.optimizers.legacy` exists, maybe code would be cleaner then...

Reply:
good catch! done